hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.tsx

```diff
@@ -258,7 +269,7 @@ export const PendingTables = () => {
                                 }}
```

Comment:
From line " {pendingOrder.status !== "Ready for Pickup" &&", this condition should include your modal.

Reply:
Otherwise, the pendingOrder id you passed into the modal will be incorrect.

Reply:
Fixed